MediaSystem.cpp

```diff
@@ -117,6 +120,10 @@ class WideVine : public IMediaKeys, public widevine::Cdm::IEventListener
         Config config;
         config.FromString(configline);
 
+        if (config.Keybox.IsSet() == true) {
+            Core::SystemInfo::SetEnvironment("WIDEVINE_TEST_KEYBOX_PATH", config.Keybox.Value().c_str());
```

Comment:
Can we leave out the _TEST_? If I am not mistaken, for us, as metrological it is a test keybox (not a production targeted keybox) however, If I am not mistaken, this is also required if we do it for production. Than the _TEST_ in there would be a bit misleading.
Also was wondering if it indicates a path or a full filname ?If it is the filename I would expect WIDEVINE_KEYBOX_FILELOCATOR (or whatever, as long as it depicts FILE and not PATH)

Reply:
The `WIDEVINE_TEST_KEYBOX_PATH` has been changed to `WIDEVINE_KEYBOX_PATH`. The environment variables indicates a path to a keybox, so the `_PATH` suffix is justified.

Reply:
Keep in mind that this change also required a slight fixup on the receiving end. PR [available here](https://github.com/Metrological/widevine/pull/3)